js/data/pokedex.js

```diff
@@ -10579,5 +10579,75 @@ var POKEDEX_XY = $.extend(true, {}, POKEDEX_BW, {
     },
     "w": 305.0,
     "ab": "Aura Break"
+  },
+  "Zygarde-Cell": {
+    "t1": "Dragon",
+    "t2": "Ground",
+    "bs": {
+      "hp": ,
```

Comment:
I left it blank so I could just add the actual info when its released, this shouldn't be merged any time anyways but I could fix it if you prefer for some reason?